docs/css-prop.md

```diff
@@ -6,7 +6,30 @@ The primary way to style elements with emotion is the `css` prop. It provides a
 
 ## Get Started
 
-#### Set the jsx pragma at the top of your source file.
+There are 2 ways to get started with the `css` prop.
+
+- [Babel Preset](#babel-preset)
+- [JSX Pragma](#jsx-pragma)
+
+#### Babel Preset
+
+This method will **not** work with [Create React App](https://github.com/facebook/create-react-app) or other projects that do not allow custom babel configurations.
+Use the [JSX Pragma](#jsx-pragma) method instead.
+
+**.babelrc**
+
+```json
+{
+  "presets": ["@emotion/babel-preset-css-prop"]
+}
+```
+
+> [Full `@emotion/babel-preset-css-prop` documentation](https://emotion.sh/docs/@emotion/babel-preset-css-prop)
+
+#### JSX Pragma
+
+Set the jsx pragma at the top of your source file that uses the `css` prop.
```

Comment:
Maybe this should also say you also have to import it since you don't have to in the babel preset?

Reply:
It says this in the next line. Should I move it?

Reply:
nah, didn't see it in it's whole, lgtm.